ArduCopter/config.h

```diff
@@ -332,6 +332,14 @@
 
 //////////////////////////////////////////////////////////////////////////////
 
+//////////////////////////////////////////////////////////////////////////////
+// Weathervane - allow vehicle to yaw into wind
+#ifndef WEATHERVANE_ENABLED
+# define WEATHERVANE_ENABLED !HAL_MINIMIZE_FEATURES
+#endif
```

Comment:
Better to shove this in the library.

Reply:
It's only enable/disabled in Copter.  Plane will have it as default.  It is a 'fixed' feature in quadplane as it were and I can't see why anyone would want to switch it off.  Does that sound correct to you?